### ISSUE TITLE
Uninstallation bug

### ISSUE THREAD

**Issue by Malachov** [swyddfa/esbonio#332]
I tried esbonio. I uninstalled it in VS Code extensions, and now exstension Restructured text is not working showing this.
![image](https://user-images.githubusercontent.com/39381691/158371426-cd832178-de39-42aa-8f9f-5eafc2179d0b.png)

There are also three lines with esbonio settings automatically added

In VS Code extensions output there is still Esbonio Language server.

I checked pip list and esbonio is not installed

I'm using:

reStructuredText: v178.0.0
VS Code insiders: 1.66.0

**Comment by lextm** [comment]
This is by design. Now esbonio language server is mandatory if you use vscode-restructuredtext extension. If you uninstall esbonio, all kinds of messages/errors are expected.

**Comment by Malachov** [comment]
It's OK with messages, but problem is that restructured text is not working anymore (it was working before esbonio install). I also uninstalled and installed resturctured text again (I would suspect, that esbonio should be installed then if it's compulsory), but still not working. If i install both of them. Then there are two icons of preview 
![image](https://user-images.githubusercontent.com/39381691/158889809-a458b79a-fac9-41b3-be31-b11c82c946ed.png)
Unfortunately none of them working

If I use the one for restructured text, it shows
![image](https://user-images.githubusercontent.com/39381691/158889869-e81cc041-f959-4a03-b225-95af7f6e94cd.png)


Finally somehow I got error that probably gives me more necessary information. It seems that there are two extensions in parallel maybe after reinstalling
![image](https://user-images.githubusercontent.com/39381691/158889681-f54d4bfb-353b-47cb-95c0-9748c3479508.png)

I don't know much about how VS Code extensions works, but if i checked `myuser\.vscode-insiders\extensions` folder with extensions, I found just on and file structure seems OK
![image](https://user-images.githubusercontent.com/39381691/158890484-30f45d50-08d1-4f58-be31-c336b8184046.png)

**Comment by alcarney** [comment]
Are you on Windows?

I tried the preview in the latest version of the `swyddfa.esbonio` extension on Windows today and unfortunately it's broken because of #340  and  #341.

I'm not sure what the state of the preview is in the vscode-restructuredtext extension is, but 0.10.0 of the language server just introduced a new way of doing previews so unfortunately there could be some disruption there....

**Comment by lextm** [comment]
> It seems that there are two extensions in parallel maybe after reinstalling

That's a serious problem and you must install only one of them. I might change vscode-restructuredtext to automatically help uninstall conflicting extensions in the future.

**Comment by Malachov** [comment]
Hello. Yes I'm on windows, but I tried to show preview on wsl version and also no success (though I didmt test more settnings or reinstall there). I think there is some collision in some shared sorces as when i run Restructured text preview, it give me notation from esbonio and also change local settimgs for esbonio even when it is uninstalled.

If it would be hard to replicate, just text me how to share logs etc. I checked vs code output of esbonio and esbonio language server, but havent found a clue, only message that it's not working prolerly. One of them still in outputs avfter uninstallation.
